### PR TITLE
ZFIN-8380: Antibody Edit Page Updates

### PR DESCRIPTION
**Pull request by rtaylorzfin**
Allow name change on antibody edit page.

### CODE REVIEW

File: source/org/zfin/antibody/Antibody.java [hunk 1]
```diff
@@ -37,6 +38,11 @@ public class Antibody extends Marker {
     @JsonView(View.AntibodyDetailsAPI.class)
     private List<Marker> antigenGenes;
 
+    @JsonView(View.AntibodyDetailsAPI.class)
+    @JsonProperty("abregistryID")
+    @Transient
+    private String ABRegistryID;
```
Comment (cmpich) [inline source/org/zfin/antibody/Antibody.java:44]: Can we use lower case for this java attribute?

File: test/org/zfin/antibody/api/AntibodyDetailsControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+        List<String> attributions = service.getPublicationAttributionsForAntibodyAntigenGeneRelationship(zdbID, "alcamb");
+        assertEquals(2, attributions.size());
+
+        assertTrue(attributions.contains("ZDB-PUB-111111-1"));
+        assertTrue(attributions.contains("ZDB-PUB-111110-1"));
+    }
+
+    @Test
+    public void getAntigenGeneRelationshipo() {
```
Comment (cmpich) [inline test/org/zfin/antibody/api/AntibodyDetailsControllerTest.java:103]: typo with the last character of the method name?

File: source/org/zfin/antibody/Antibody.java (hunk 1 above)
Comment (rtaylorzfin) [reply source/org/zfin/antibody/Antibody.java]: Yes, good suggestion.

File: test/org/zfin/antibody/api/AntibodyDetailsControllerTest.java (hunk 2 above)
Comment (rtaylorzfin) [reply test/org/zfin/antibody/api/AntibodyDetailsControllerTest.java]: Good catch

**Pull request merged**